source/style-guide/style/metadata.txt

```diff
@@ -0,0 +1,19 @@
+.. _metadata:
+
+=============
+Metadata
+=============
+
+.. default-domain:: mongodb
+
+.. include:: /includes/styles/corrections.rst
+
+Use ``.. meta::`` to add meta tags to a page. Adding meta tags aids in 
+searches both internally and via the Internet.
+
+.. code-block:: rst
+     
+   .. meta::
+      :description: read concern, local read concern, read isolation, 
```

Comment:
I feel like we should explain what `:description:` means here. Is it required? What is its purpose?